cmd/earth/main.go

```diff
@@ -1158,6 +1165,19 @@ func (app *earthApp) actionRegister(c *cli.Context) error {
 		pword = string(enteredPassword)
 	}
 
+	var interactiveAccept bool
+	if !app.termsConditionsPrivacy {
+		// TODO: add link when we have one
+		rawAccept := promptInput("Do you accept Earthly's T&Cs <link> and Privacy Policy <link>? [y/N]")
+		if rawAccept == "" {
+			rawAccept = "n"
+		}
+		accept := strings.ToLower(rawAccept)[0]
+
+		interactiveAccept = accept == 'y'
```

Comment:
I like how this will work with "yup", "yeah", "yar", and "yucky".

Reply:
Yeah I didn't want to whitelist affirmative answers so we are here. I can hardcode it to yes if you want

Reply:
might be best to check for "y" or "yes". But I don't feel strongly one way or the other.